frontend/components/ListContainer.vue

```diff
@@ -1,10 +1,9 @@
 <template>
   <div class="wrapper">
-    <searchBar v-if="!isMobile()" class="searchBar" />
-    <FilterContent class="filterBtn" />
+    <searchBar class="searchBar" />
```

Comment:
SearchBar component is sometimes called searchbar in places like this. 
Everywhere else those elements are named in PascalCase.

Reply:
Fixed.